src/components/Data/Collections/Update.vue

```diff
@@ -56,9 +54,11 @@ export default {
       )
     },
     mappingAttributes() {
-      return this.collection
-        ? omit(this.collection.mapping, '_kuzzle_info')
+      const mapping = this.collection
+        ? omit(this.collection.mapping, '_kuzzle_info', 'dynamic')
         : null
+      const dynamic = mapping.dynamic
+      return { dynamic: dynamic, properties: mapping }
```

Comment:
It seems not possible to get `mapping.dynamic` after omit `dynamic` 
Also, `mapping.dynamic` will lead to an error if `this.collection` is null